rules.md

```diff
@@ -4,6 +4,40 @@ See below all rules that can be used for monorepo linting.
 
 Those configurations should be added to a file in the root of the monorepo called '.monolint.json'. If you create this file in intermediate folder (or even in the module folder), it will be merged to the root and default configurations also.
 
+## **module-required-files**
```

Comment:
It seems the pipeline is failing because this file needs to be updated

Reply:
Done! Thanks.!